content/docs/authorization/Migration guide from v1 to v2/_index.md

```diff
@@ -5,7 +5,9 @@ weight: 1
 description: >
   CSM for Authorization v1 to v2 Migration Guide
 ---
-
+{{% pageinfo color="primary" %}}
```

Comment:
Why do we need the quay message in this page?

Reply:
I will remove this. Its not required